src/test/methods/upsert.ts

```diff
@@ -121,13 +121,12 @@ test.serial('increment and set with upsert', async t => {
 		Key: {
 			id: '5'
 		},
-		UpdateExpression: 'SET #k_foo=:v_foo, #k_salary=if_not_exists(#k_salary, :_v_empty_value)+:v_salary',
+		UpdateExpression: 'SET #k_foo=:v_foo ADD #k_salary :v_salary',
```

Comment:
Can you add an extra property to the `$inc` operator? Just to see how it would work with multiple values.

Reply:
done